terraform/aws/components/nhais/locals_env_variables.tf

```diff
@@ -46,7 +46,8 @@ locals {
     },
     {
       name = "NHAIS_MESH_HOST"
-      value = var.nhais_mesh_host //https://fake-mesh:8829/messageexchange/ / https://msg.intspineservices.nhs.uk/
+      // TODO: hardcoded default for now but it should just be from the var. This will be different for every env
+      value = var.nhais_mesh_host == "" ? "https://mesh.vp.nhsredteam.internal.nhs.uk:8829/messageexchange/" : var.nhais_mesh_host
```

Comment:
make the url depend on environment, replace vp with var.environment

Reply:
fake-mesh should only ever be deployed into the vp cluster. I suppose this default value shouldn't be here at all and it should be specified at deployment.

Reply:
actually, if we want to use a default it should be in the variables.tf right?

Reply:
we have a default in var.nhais_mesh_host, which is empty string.
I'm aware that fake-mesh should exist only in vp, but the component code should be environment agnostic.
All environent wise settings should be done in etc/*.tfvars - deploy fake-mesh for vp, do not for others